db/dump_manage.py

```diff
@@ -101,7 +104,8 @@ def _json_highlevel(location, rotate):
 @click.option("--id", type=int)
 @click.option("--threads", "-t", type=int)
 def incremental(location, id, threads):
-    db.init_db_engine(config.SQLALCHEMY_DATABASE_URI)
+    # inits the db engine
+    create_app()
```

Comment:
Does it feel a bit weird to you to create the full web app just to init the db engine?

Reply:
The idea here was to have only one place in the codebase that calls `db.init_db_engine`. The advantage of having it as part of the flask init is that we use flask's `app.config` inside `create_app`, which allows us to do things like override the database uri with the test config file. To do the same in `manage.py` we would have to specifically know which config file to import in this file.
Note that critiquebrainz does something similar (but not exactly the same) - https://github.com/metabrainz/critiquebrainz/blob/c7033487c5ce1beb45703193d47b90f4ec357ad9/manage.py#L98

Reply:
Ah okay, makes sense.